components/browser/menu/src/test/java/mozilla/components/browser/menu/item/BrowserMenuHighlightableItemTest.kt

```diff
@@ -59,14 +59,16 @@ class BrowserMenuHighlightableItemTest {
 
         val view = inflate(item)
 
-        val textView = view.findViewById<TextView>(R.id.text)
+        val textView = view.findViewById<TextView>(R.id.imageText)
         assertEquals(textView.text, "label")
 
-        val imageView = view.findViewById<AppCompatImageView>(R.id.image)
+        val image = textView.compoundDrawables[0]
+
         val highlightImageView = view.findViewById<AppCompatImageView>(R.id.highlight_image)
 
-        assertNotNull(imageView.drawable)
-        assertNotNull(imageView.imageTintList)
+        assertNotNull(image)
+        // TODO: Not sure how to verify iconTintColorResource:
```

Comment:
Any ideas on how to best test this? I'm having trouble since both color filter and compoundDrawableTintList are null